packages/syft/src/syft/service/code/user_code.py

```diff
@@ -709,10 +733,10 @@ class SubmitUserCode(SyftObject):
     __repr_attrs__ = ["func_name", "code"]
 
     @property
-    def kwargs(self) -> List[str]:
+    def kwargs(self) -> Optional[dict[Any, Any]]:
```

Comment:
check if this could be `Optional[dict[str, Any]]`

Reply:
I don't think so @kiendang since in the code there are already multiple places that it's defined as `Optional[Dict[Any, Any]]`
Also in line 410 of `user_code.py`, we use it with `[x.verify_key for x in self.input_policy_init_kwargs.keys()]`

Reply:
ah I see